flexget/components/sites/sites/passthepopcorn.py

```diff
@@ -82,24 +76,6 @@
 RELEASE_TYPES = {'non-scene': 0, 'scene': 1, 'golden popcorn': 2}
 
 
-@db_schema.upgrade('passthepopcorn')
-def upgrade(ver, session):
-    if ver is None:
-        ver = 0
-    if ver == 0:
-        raise db_schema.UpgradeImpossible
-    return ver
```

Comment:
I guess technically a new version should be added that drops the cookie table. We do have a utility function to do that dropping. https://github.com/Flexget/Flexget/blob/ba878634b022cb5e94d32e4d2e593e38a19714a4/flexget/utils/sqlalchemy_utils.py#L98

Reply:
Seems like a better idea than abandoning it.. I will try and re-insert something.

Reply:
I THINK? I fixed it the old code went up to version 0 so I am only doing a table check once and removing it if it exists.. Let me know if I got this right in the new commit.

Reply:
Going to stop playing with the import upgrade I am guessing it is fine? Is actually the expected outcome? not sure why it shows as FAILED however then?

`FAILED flexget/tests/test_migrate.py::TestMigrate::test_upgrade - ValueError: The plugin passthepopcorn has no stored schema to reset. `

Reply:
I think we need to do a full 'upgrade' i.e. change the current schema version to 2 in the `Base` declaration, then we can remove the old upgrades and change the if in the upgrade function to do `if ver is None or ver < 2` in order to capture all previous states. Inside that block you'll do the table drop.

Reply:
We need the `ver is None or ver < 2` to prevent that current error of the tests, and you were right to have the `ver=2` in there.

Reply:
Did a little more reading, don't think we can call UPGRADE unless we are declaring a NEW base schema but since what we WANT to do is clean up and remove the table  I think we want to call reset_schema instead of upgrade.

Reply:
I have removed all the upgrade code and DB import references as well as the plugin isn't using the db at all now.. If you have a way to clean up all tables I am all ears (probably will need to import the DB libraries again) but as it stands it works without the upgrade code.

Reply:
Oh, I realized I was complicating things a bit. To clean up the db we can just put the function in an `@event('manager.upgrade')` rather than the `@db_schema.upgrade`. That way we can leave all the other DB stuff you already cleaned up totally out still.

Reply:
> To clean up the db we can just put the function in an @event('manager.upgrade') rather than the @db_schema.upgrade. That way we can leave all the other DB stuff you already cleaned up totally out still.

I don't see any existing plugins that do this, also in order to have the table search and drop calls I would have to import the schema and DB functions still..

Looking into my SQL lite DB this table should have next to nothing in it and only exist for a small number of users.

I would prefer to abandon it and move on.